pkg/sanity/controller.go

```diff
@@ -1360,6 +1275,22 @@ var _ = DescribeSanity("Controller Service [Controller Server]", func(sc *TestCo
 		})
 	})
 
+	Describe("ControllerVolumeLifecycle", func() {
```

Comment:
"ControllerVolumeLifecycle" is plain English, not the name of a gRPC call as in the other `Describe` cases. Better just use "volume lifecycle". "Controller" can be left out because the top-level `Describe` already is "Controller Service [Controller Server]" - no need to repeat it here.